apps/root/src/constants/addresses.ts

```diff
@@ -474,6 +476,7 @@ export const COMPANION_ADDRESS: AddressMap<PositionVersions> = {
     [NETWORKS.moonbeam.chainId]: '0x6C615481E96806edBd9987B6E522A4Ea85d13659',
     [NETWORKS.baseGoerli.chainId]: '0x6C615481E96806edBd9987B6E522A4Ea85d13659',
     [Chains.ROOTSTOCK.chainId]: '0x5872E8D5Ec9Dbf67949FdD4B5e05707644D60876',
+    [NETWORKS.avalanche.chainId]: '0x6C615481E96806edBd9987B6E522A4Ea85d13659',
```

Comment:
Sir this is not the correct companion address for AVAX, ask the BE guys if you need a reference, or use the Addresses doc. Lets be mindfull of details ⭐